src/zpragma/zpragma.c

```diff
@@ -191,6 +195,51 @@ void write_redirect(char *sname, char *value)
     fclose(fp);
 }
 
+void data_from_file(char *line){
```

Comment:
Is this still needed?

Reply:
No it is not needed, pragma data_from_file was from before I moved the brany